doc/nrf/images/thread_platform_design_multi.svg

```diff
@@ -0,0 +1,330 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@hubertmis I'm not sure how shall this second "zephyr integration " be placed. Should it extend above MPSL as well?
"zephyr integration" here was intended to be a common name for all the shim layers and any other zephyr components.

Reply:
I have some comments about this picture:

1. MPSL shall not touch OpenThread, because OpenThread does not use any MPSL APIs.
2. MPSL is a dependency of 802.15.4 driver and BLE Ctrl and as a dependency shall be placed below (not at the same level). However, both BLE Ctrl and 802.15.4 shall still have access to the HW.
3. 802.15.4 driver does not use Timeslot API, but if MPSL were moved below 802.15.4 (2.), the arrow and its label could be removed.
4. There are some NCS modules (not Zephyr modules) used to configure 802.15.4 driver and MPSL that are missing in the image, but I don't think it would be good to obfuscate the image with such details. I would leave this part as it is.
5. To answer your question: I would leave the 2nd "Zephyr integration" as it is, skipping NCS configuration modules.